src/librustdoc/html/markdown.rs

```diff
@@ -1365,6 +1368,11 @@ fn init_id_map() -> FxHashMap<String, usize> {
     map.insert("trait-implementations".to_owned(), 1);
     map.insert("synthetic-implementations".to_owned(), 1);
     map.insert("blanket-implementations".to_owned(), 1);
+    map.insert("associated-types".to_owned(), 1);
+    map.insert("associated-const".to_owned(), 1);
+    map.insert("required-methods".to_owned(), 1);
+    map.insert("provided-methods".to_owned(), 1);
+    map.insert("implementors".to_owned(), 1);
     map
```

Comment:
I think you forgot to add `methods` and `synthetic-implementors` to this list.

Reply:
There is no `methods` ID, only a class. Adding `synthetic-implementors`.

Reply:
Ah, okay 👍